src/CC-BY-NC-ND-3.0-IGO.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="CC-BY-NC-ND-3.0-IGO"
+            name="Creative Commons Attribution-NonCommercial-NoDerivs 3.0 IGO" listVersionAdded="3.10">
```

Comment:
Just for purposes of aligning with the titles as they currently appear on the license list -- can this be changed to the following?

`Creative Commons Attribution Non Commercial No Derivatives 3.0 IGO`

I recognize that that's a slightly different formatting than the way it appears in the license itself, but I'm guessing for the license list purposes it's probably better to align with the way the others are currently displayed.

Reply:
All fixed! Thanks for catching that!